tmc-prototype/CentralNode/CentralNode/CentralNode.py

```diff
@@ -36,164 +31,91 @@ class CentralNode(SKABaseDevice):
+                     + CONST.STR_UNKNOWN
                 else:
-                    print "Subarray Health state event returned unknown value! \n", evt
-                    self._read_activity_message = "Subarray Health state event returned unknown value! \n" + str(evt)
+                    print CONST.STR_HEALTH_STATE_UNKNOWN_VAL, evt
+                    self._read_activity_message = CONST.STR_HEALTH_STATE_UNKNOWN_VAL + str(evt)
                 # Aggregated Health State
                 failed = 0
                 degraded = 0
                 unknown = 0
-                ok = 0
-                for value in (self.subarrayHealthStateMap.values()):
-                    if value == 2:
+                _ok = 0
+                for value in self.subarrayHealthStateMap.values():
+                    if value == CONST.ENUM_FAILED:
                         failed = failed + 1
                         break
-                    elif value == 1:
+                    elif value == CONST.ENUM_DEGRADED:
                         self._telescope_health_state = 1
                         degraded = degraded + 1
-                    elif value == 3:
-                        self._telescope_health_state = 3
+                    elif value == CONST.ENUM_UNKNOWN:
+                        self._telescope_health_state = CONST.ENUM_UNKNOWN
                         unknown = unknown + 1
 
                     else:
-                        self._telescope_health_state = 0
-                        ok = ok + 1
+                        self._telescope_health_state = CONST.ENUM_OK
+                        _ok = _ok + 1
 
-                if ok == len(self.subarrayHealthStateMap.values()):
-                    self._telescope_health_state = 0
+                if _ok == len(self.subarrayHealthStateMap.values()):
+                    self._telescope_health_state = CONST.ENUM_OK
 
                 elif failed != 0:
-                    self._telescope_health_state = 2
+                    self._telescope_health_state = CONST.ENUM_FAILED
 
                 elif degraded != 0:
-                    self._telescope_health_state = 1
+                    self._telescope_health_state = CONST.ENUM_DEGRADED
 
                 else:
-                    self._telescope_health_state = 3
+                    self._telescope_health_state = CONST.ENUM_UNKNOWN
 
-            except Exception as e:
-                print "Unexpected error while aggregating Health state!\n", e
-                self._read_activity_message = "Unexpected error while aggregating Health state!\n" + str(e)
-                self.devlogmsg("Unexpected error while aggregating Health state!", 2)
+            except Exception as except_occured:
+                print CONST.ERR_AGGR_HEALTH_STATE, except_occured
+                self._read_activity_message = CONST.ERR_AGGR_HEALTH_STATE + str(except_occured)
+                self.devlogmsg(CONST.ERR_AGGR_HEALTH_STATE, 1)
```

Comment:
Replace 1 with constant value.

Reply:
Replaced magic numbers with respective TANGO Log Levels in Central Node.

Reply:
Updated log level nos to corresponding enum label for SubarrayNode, DishLeafNode, and DishMaster.

Reply:
Updated log level nos to corresponding enum label for CentralNode. Commit id is 8b0d886abad93b916fa40d3d325d9e31083eedb2.